### PR TITLE
Order summary

### PR DESCRIPTION
**Pull request by mohammed-sas**
Screen Shot 2022-04-02 at 11 44 59 PM
Screen Shot 2022-04-02 at 11 44 30 PM
### Features
1.user can select delivery address from the saved addresses
2.user can view the summary of products ordered
3.user can view the overall summary of the order

### CODE REVIEW

**Review by Aazam10 — APPROVED** [review]
Great work

File: src/pages/order summary/OrderSummary.jsx [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+  return (
+    <main className={classes["summary-container"]}>
+      <div className={classes["container"]}>
+        <div className={classes["address-container"]}>
+          <h3>Delivery Address</h3>
+          <div>
+            {selectedAddress ? (
+              <MiniAddressCard
+                address={selectedAddress}
+                showInput={!!selectedAddress}
```
Comment (starver20) [inline src/pages/order summary/OrderSummary.jsx:24]: Why double negation?

**Review by starver20 — APPROVED** [review]
Neatly coded, good to merge

File: src/pages/order summary/OrderSummary.jsx (hunk 1 above)
Comment (mohammed-sas) [reply src/pages/order summary/OrderSummary.jsx:24]: To convert the value into boolean

**Pull request merged**